ui/src/errors/components/Error401.jsx

```diff
@@ -4,10 +4,12 @@ import ErrorPage from './ErrorPage';
 
 class Error401 extends Component {
   render() {
+    const previousURL = document.referrer;
```

Comment:
is it always there?

Reply:
If the page was entered directly (like by pasting http://inspirehep.net/errors/404) and there will be no referrer, it will return empty string and the page will look like it used to look previously without redirect url (lack of referrer won't cause the page to break :) ).
<img width="909" alt="Screenshot 2023-01-25 at 17 03 05" src="https://user-images.githubusercontent.com/55505399/214613457-7d6beb26-cb9c-40dd-929b-ee0177005939.png">